src/renderer/features/sidebar/components/sidebar-playlist-list.tsx

```diff
@@ -161,8 +185,43 @@ export const SidebarPlaylistList = ({ data }: SidebarPlaylistListProps) => {
         [handlePlayQueueAdd],
     );
 
+    const handleAddCurrent = useCallback(
+        (playlist: Playlist) => {
+            if (!currentSong) return;
+
+            addToPlaylistMutation.mutate(
+                {
+                    body: { songId: [currentSong.id] },
+                    query: { id: playlist.id },
+                    serverId: serverID,
+                },
+                {
+                    onError: (err) => {
+                        toast.error({
+                            message: `[${playlist?.name}] ${err.message}`,
+                            title: t('error.genericError', { postProcess: 'sentenceCase' }),
+                        });
+                    },
+                },
+            );
+
+            toast.success({
+                message: t('form.addToPlaylist.success', {
+                    message: 1,
+                    numOfPlaylists: 1,
+                    postProcess: 'sentenceCase',
+                }),
+            });
+        },
+        [addToPlaylistMutation, currentSong, serverID, t],
+    );
+
     const memoizedItemData = useMemo(() => {
-        const base = { defaultFullPlaylist, handlePlay: handlePlayPlaylist };
+        const base = {
+            defaultFullPlaylist,
+            handleAdd: handleAddCurrent,
```

Comment:
This should not be enabled for Navidrome shared playlists (only the owner can change them)